client/v3/watch.go

```diff
@@ -855,6 +855,9 @@ func (w *watchGRPCStream) serveSubstream(ws *watcherStream, resumec chan struct{
 				nextRev = wr.Events[len(wr.Events)-1].Kv.ModRevision + 1
 			}
 
+			if nextRev < ws.initReq.rev {
+				fmt.Println("oops --------> initReq.rev(", ws.initReq.rev, ") vs nextRev(", nextRev, ")")
```

Comment:
Please also print whether there is any Events in the watch response. If yes, print the biggest `ModRevision`?

Reply:
Will update it.